README.md

```diff
@@ -6,7 +6,7 @@
 
 Install the `aiidalab_widgets_base` python package via:
 ```
-pip install aiidalab-widgets-base 
+pip install aiidalab-widgets-base
```

Comment:
... although it nicely caught this useless trailing space 😅

Reply:
Yeah, I'd prefer to lint the markdown files as well (see comment above).